lib/lotus/generators/application/container/Gemfile.tt

```diff
@@ -12,8 +12,9 @@ gem 'lotus-view',        require: false, github: 'lotus/view'
 gem 'lotus-model',       require: false, github: 'lotus/model'
 gem 'lotusrb',                           github: 'lotus/lotus'
 <%- else -%>
-gem 'lotusrb',      '<%= Lotus::VERSION %>'
-gem 'lotus-model', '<%= config[:lotus_model_version] %>'
+gem 'lotusrb',        '<%= Lotus::VERSION %>'
+gem 'lotus-model',    '<%= config[:lotus_model_version] %>'
+gem 'lotus-helpers',  '<%= config[:lotus_helpers_version] %>'
```

Comment:
can we remove 1 space in between `gem 'lotus-helpers',` and `'<%= config[:lotus_helpers_version] %>'`?